docs/how_to/documenting_new_feature_toggles.rst

```diff
@@ -0,0 +1,74 @@
+
+``toggle_implementation``
+-------------------------
+
+Must be one of the following:
+
+- "WaffleFlag", "WaffleSwitch", "CourseWaffleFlag": for objects that inherit from ``waffle_utils``' ``WaffleFlag``, ``WaffleSwitch`` or ``CourseWaffleFlag``, respectively.
+- "ExperimentWaffleFlag": this is for objects that inherit from ``lms.djangoapps.experiments.flags.ExperimentWaffleFlag``. See `this documentation page <https://openedx.atlassian.net/wiki/spaces/AC/pages/1250623700/Bucketing+users+for+an+experiment>`__ and the `reference implementation <https://github.com/edx/edx-platform/blob/master/lms/djangoapps/experiments/flags.py#L21>`__.
+- "ConfigurationModel": for objects that inherit from `config_models.models.ConfigurationModel <https://github.com/edx/django-config-models/>`__.
+- "DjangoSetting": feature toggles can also be enabled by defining a `Django setting <https://docs.djangoproject.com/en/dev/topics/settings/>`__.
+
+``toggle_use_cases``
+--------------------
+
+To decide what are the use cases of a feature toggle, one should refer to the `list of use cases outlined in OEP-17 <https://open-edx-proposals.readthedocs.io/en/latest/oep-0017-bp-feature-toggles.html#use-cases>`__:
+
+Temporary (``toggle_use_cases = temporary``):
+
+* Use Case 1: Incremental Release (expected timeline: 3 months)
+* Use Case 2: Launch Date (expected timeline: 3 months)
+* Use Case 3: Ops - Monitored Rollout (expected timeline: 3 months)
+* Use Case 5: Beta Testing (expected timeline: 6 months)
+
+(Semi-)Permanent:
+
+* Use Case 4: Ops - Circuit Breaker (expected timeline: 5 years, ``toggle_use_cases = circuit_breaker``)
+* Use Case 6: VIP / White Label (expected timeline: 5 years, ``toggle_use_cases = vip``)
+* Use Case 7: Opt-out/Opt-in (expected timeline: 2 years, ``toggle_use_cases = opt_out`` or ``toggle_use_cases = opt_in``)
+* Use Case 8: Open edX option (expected timeline: 3 years, ``toggle_use_cases = open_edx``)
+
+Examples
+========
+
+Boilerplate template
+--------------------
+
+Copy-paste this boilerplate template to document a feature toggle::
+
+    # .. toggle_name: SOME_FEATURE_NAME
+    # .. toggle_implementation: WaffleFlag OR WaffleSwitch OR CourseWaffleFlag OR ExperimentWaffleFlag OR ConfigurationModel OR DjangoSetting
+    # .. toggle_default: True OR False
+    # .. toggle_description: Add here a detailed description of the consequences of enabling this feature toggle.
+    #   Note that all annotations can be spread over multiple lines by prefixing every line after the first by
+    #   at least two spaces, plus the leading space.
+    # .. toggle_warnings: Add here additional instructions that users should be aware of. For instance, dependency
+    #   on additional settings or feature toggles should be references here.
+    # .. toggle_use_cases: temporary OR circuit_breaker OR vip OR opt_out OR opt_in OR open_edx
+    # .. toggle_creation_date: 2020-12-31
+    # .. toggle_expiration_date: 2050-12-31
```

Comment:
* Rename 
* Can you update example date to creation data + 6 months?
* What if I have a permanent toggle?  Should I leave this blank or omit this?  Should I add the expected lifetime to return to this?

Reply:
:ok_hand: